Yank/tests/test_yank.py

```diff
@@ -31,6 +32,7 @@
 from yank.repex import ThermodynamicState
 from yank.pipeline import find_components
 
+
 from yank.yank import *
```

Comment:
Does it cause problems with python 3? Because I find it very convenient to use it (only) in tests since in `test_yank.py` we want to test every single public element contained in `yank.py`.